src/Command/DumpMetadataCommand.php

```diff
@@ -26,20 +27,24 @@ final class DumpMetadataCommand extends Command
 
     protected static $defaultName = 'app:metadata:dump';
     private $dumper;
+    private $url;
 
-    public function __construct(Dumper $dumper)
+    public function __construct(Dumper $dumper , string $url )
     {
         parent::__construct();
         $this->dumper = $dumper;
+        $this->url = $url;
     }
 
     protected function configure()
     {
+        $this->addArgument('url', InputArgument::OPTIONAL, 'what is the url of the repo you want to mirror ?');
         $this->setDescription('Dump metadata files.');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $url = $input->getArgument('url') ? $input->getArgument('url') : $this->url;
```

Comment:
could you please use `addArgument` default value. https://github.com/symfony/symfony/blob/4.0/src/Symfony/Component/Console/Command/Command.php#L367

Reply:
@IndraGunawan I've resolved this , can you please merge it ?